cosmrs/Cargo.toml

```diff
@@ -12,7 +12,7 @@ edition = "2021"
 rust-version = "1.56"
 
 [dependencies]
```

Comment:
Kept getting a compilation error here, needed to update the local package version